content/attacks/posts/2023-04-28-0vix.md

```diff
@@ -0,0 +1,41 @@
+---
+date: 2023-04-28
+target-entities: 
+- 0vix
+entity-types:
+- DeFi
+- Lending
```

Comment:
Rewrite as `Lending Platform`, since it will lead to duplicated in the entity types